packages/main/src/Breadcrumbs.ts

```diff
@@ -413,26 +376,12 @@ class Breadcrumbs extends UI5Element {
 			shiftKey,
 		}, true)) {
 			e.preventDefault();
+			return;
 		}
-	}
 
-	_onLabelPress(e: MouseEvent | KeyboardEvent) {
-		const items = this._getItems(),
-			item = items[items.length - 1],
-			{
-				altKey,
-				ctrlKey,
-				metaKey,
-				shiftKey,
-			} = e;
-
-		this.fireEvent<BreadcrumbsItemClickEventDetail>("item-click", {
-			item,
-			altKey,
-			ctrlKey,
-			metaKey,
-			shiftKey,
-		});
+		if (item._isCurrentPageItem) {
```

Comment:
it would be nice to be able to prevent the page refresh, perhaps keep the event and check if it was prevented?

Reply:
yes, currently the "item-click" event is fired before we refresh the page.. So, if the app prevented the default action in their listener to that event, the page refresh will also be prevented.... 

I just re-tested this on the sample ui5-breadcrumb with id="breadcrumbsPreventDefault" on the test page Breadcrumbs.html

Or do I misunderstand your concern... I'm available to look at it further..

Reply:
right, sorry. diff made it seem like the event was removed and I had to expand more. all good then.

Reply:
thanks for re-checking..